src/components/fee-slider/fee-slider.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Text } from 'react-native';
+
+import { Slider } from '../slider/slider';
+import { StyledTextInput } from '../styled-text-input/styled-text-input';
+
+export const FeeSlider = () => {
+  // TODO: replace with NumberInput
+  const [value, setValue] = useState(0);
+  return (
+    <>
+      <Text>Gas Fee</Text>
```

Comment:
remove this label from here, it should not be related to the shared component.  
You should create such components: 
1. `Slider` (what you did) already
2. `StyledInputSlider` = `StyledTextInput` + `Slider`
3. `FormInputSlider` = `StyledInputSlider` + `ErrorMessage` (that should be used inside forms)

Then you will use
```
<Text> Gas Fee </Text>
<FormInputSlider name="gasFee" />
```
inside `send-bottom-sheet.tsx`